### PR TITLE
Fix geographic subject mapping error.

### LINKED ISSUE #2275: Subject geographic mapping error
For druid:hf760px4366:

Original document:
``` 
<subject altRepGroup="04" authority="lcsh">
    <geographic>&#x1E32;iryat &#x1E24;ayim (Haifa, Israel)</geographic>
  </subject>
  <subject altRepGroup="04" authority="lcsh">
    <geographic>&#x5E7;&#x5E8;&#x5D9;&#x5EA; &#x5D7;&#x5D9;&#x5D9;&#x5DD;</geographic>
  </subject>
```

```
Best guess roundtripped node:
<subject>
  <geographic authority="lcsh">Ḳiryat Ḥayim (Haifa, Israel)</geographic>
  <geographic authority="lcsh">קרית חיים</geographic>
</subject>
```

### PR DESCRIPTION
**Pull request by justinlittman**
closes #2275

## Why was this change made?
Bad code.

## How was this change tested?
Unit, sdr-deploy

## Which documentation and/or configurations were updated?
NA

### CODE REVIEW

**Comment by justinlittman** [comment]
Before:
```
Status (n=100000; not using Missing for success/different/error stats):
  Success:   99144 (99.666%)
  Different: 332 (0.334%)
  To Cocina error:     0 (0.0%)
  To Fedora error:     0 (0.0%)
  MODS normalizer error:     0 (0.0%)
  Missing (no descMetadata):     524 (0.524%)
```

After:
```
Status (n=100000; not using Missing for success/different/error stats):
  Success:   99144 (99.666%)
  Different: 332 (0.334%)
  To Cocina error:     0 (0.0%)
  To Fedora error:     0 (0.0%)
  MODS normalizer error:     0 (0.0%)
  Missing (no descMetadata):     524 (0.524%)
```

File: app/services/cocina/to_fedora/descriptive/subject.rb [hunk 1]
```diff
@@ -68,7 +68,8 @@ def write_subject(subject, subject_value, alt_rep_group: nil, type: nil, display
         end
 
         def write_parallel(subject, subject_values, alt_rep_group:, display_values: nil)
-          if subject.type == 'place'
+          # If a geographic and geographicCode, get written together.
```
Comment (ndushay) [inline app/services/cocina/to_fedora/descriptive/subject.rb:71]: this comment makes no sense to me.  Can you edit?

**Review by ndushay — APPROVED** [review]
approving, with caveat of considering changing a comment.

File: app/services/cocina/to_fedora/descriptive/subject.rb (hunk 1 above)
Comment (justinlittman) [reply app/services/cocina/to_fedora/descriptive/subject.rb:71]: Yup.

**Pull request merged**